### DISCUSSION TITLE
Getting error in f1 score, recall, precision while accuracy score worked.

### DISCUSSION THREAD

**Discussion by saleharashid01** [AammarTufail/python_ka_chilla2023#87] [Q&A] [answered]
![How to fix it](https://user-images.githubusercontent.com/121869148/213934958-082bdf37-f450-4de2-aaf2-d1d55abc3b56.jpg)
  
how to fix it
Getting error in f1 score, recall, precision while accuracy score worked. in covid dataset

**Comment by AzamBukhari** [comment, marked as answer]
AoAlaikum where ever you are gettng error the SOLUTION is a small tweak:
add this argument average='weighted'

precision_score(y_test, y_pred, average='weighted')

recall_score(y_test, y_pred, average='weighted')

f1_score(y_test, y_pred, average='weighted') 

Hope it resolve your issue, inshaaAllah

**Reply by saleharashid01** [reply]
it worked , Jazak Allah
